spec/models/bet_spec.rb

```diff
@@ -0,0 +1,92 @@
+require 'rails_helper'
+
+RSpec.describe Bet, type: :model do
+  let!(:bob) do
+    User.create(username: 'Bob', email: 'bob@mail.com', password: '123')
+  end
+  let!(:peter) do
+    User.create(username: 'Peter', email: 'peter@mail.com', password: 'pete1')
+  end
+  let!(:jack) do
+    User.create(username: 'Jack', email: 'jack@mail.com', password: 'notjack')
+  end
+
+  let(:bet1) do
+    Bet.create(bet: 'Superbowl: Atlanta vs New England, place your bets!')
+  end
+
+  let(:option1) { BetOption.create(option_text: 'Atlanta wins') }
+  let(:option2) { BetOption.create(option_text: 'New England wins') }
+
+  let!(:users_place_bets) do
+    UserBet.create(user:       bob,
+                   bet:        bet1,
+                   bet_option: option1,
+                   amount_bet: 100)
+
+    UserBet.create(user:       peter,
+                   bet:        bet1,
+                   bet_option: option1,
+                   amount_bet: 50)
+
+    UserBet.create(user:       jack,
+                   bet:        bet1,
+                   bet_option: option2,
+                   amount_bet: 300)
+  end
+
+  describe('#winner_total') do
+    subject { bet1.winner_total }
+    context 'Atlanta wins' do
+      let(:expected_atlanta_win_total) { 150 }
+      it 'returns the total amount of money bet on Atlanta winning (150)' do
```

Comment:
The number should be removed here. It could be changed and then we have to update it multiple places. 

`it 'is the correct money bet amount' ` would be enough information